lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -114,6 +119,16 @@ export class ContentMetadataComponent implements OnChanges, OnInit, OnDestroy {
     @Input()
     displayCategories = false;
 
+    /** (optional) This flag sets the metadata in read only mode
+     * preventing changes.
+     */
+     @Input()
```

Comment:
Looks like that input annotation and readOnly property are incorrectly aligned looking at rest of properties